requirements.txt

```diff
@@ -1,2 +1,3 @@
 numpy
 protobuf >= 3.20.2
+opencv-python
```

Comment:
We should make it optional.

Reply:
Done. I used [PEP-508](https://peps.python.org/pep-0508/#extras). Would that be enough?

Reply:
Will try removing the dependency from here completely.